actionpack/test/controller/mime/respond_to_test.rb

```diff
@@ -608,19 +608,29 @@ def test_invalid_format
   end
 
   def test_invalid_variant
+    logger = ActiveSupport::LogSubscriber::TestHelper::MockLogger.new
+    old_logger, ActionController::Base.logger = ActionController::Base.logger, logger
+
     @request.variant = :invalid
-    assert_raises(ActionView::MissingTemplate) do
-      get :variant_with_implicit_rendering
-    end
+    get :variant_with_implicit_rendering
+    assert_response :no_content
+    assert_equal 1, logger.logged(:info).select{ |s| s =~ /No template found/ }.size
```

Comment:
Is this consistent with how we make similar assertions elsewhere? `0 != 1` isn't going to be a terribly helpful failure message :confused: